sources/web/datalab/metadata.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/// <reference path="../../../third_party/externs/ts/node/node.d.ts" />
+/// <reference path="common.d.ts" />
+
+import fs = require('fs');
+import http = require('http');
+import logging = require('./logging');
+import url = require('url');
+
+interface Creds {
+  account: string;
+  scopes: string;
+  access_token:  string;
+  expires_in: number;
+  token_type: string;
+}
+
+interface FakeMetadata {
+  project: string;
+  project_number: string;
+  creds: Creds;
+}
+
+var metadata: FakeMetadata = {
```

Comment:
Use const or let everywhere instead of var

Reply:
Done.